deploy/gcloud/README.md

```diff
@@ -0,0 +1,112 @@
+
+The following information can be customized in setup.sh, but the default also works fine:
+
+  1. AMP_PACKAGER_VERSION_TAG. The version tag attached to your docker image when it's built and uploaded into the [Container Registry](https://cloud.google.com/container-registry).
+
+  2. AMP_PACKAGER_NUM_REPLICAS.  Number of AMP Packager replicas to run. Default is 2, you can scale up to 8 instances.
+
+  3. AMP_PACKAGER_CERT_FILENAME. The name of the amppackager certificate.
+
+  4. AMP_PACKAGER_CSR_FILENAME. The name of the amppackager [Certificate Signing Request](https://www.digicert.com/ecc-csr-creation-ssl-installation-apache.htm).
+
+  5. AMP_PACKAGER_PRIV_KEY_FILENAME. The name of the amppackager private key (see this [article])(https://www.ericlight.com/using-ecdsa-certificates-with-lets-encrypt).
+
+  Note that the cert, CSR and private key all go together. You may choose to
+  manually create these and [request the certificate from Digicert by email](https://docs.digicert.com/manage-certificates/certificate-profile-options/get-your-signed-http-exchange-certificate/). If you chose to go this route, you may simply copy these files into your
+  amppackager/deploy/gcloud/generated directory and they will not be auto-generated and
+  fulfilled by amppackager using ACME. By default, these files will be
+  automatically created given the information you supplied in setup.sh, and the
+  certificate will be requested using ACME automatically.
+
+## Run the deployment script (gcloud_up.sh)
+
+  1. Go to the directory where you installed amppackager.
+  2. cd deploy/gcloud
+  3. ./gcloud_up.sh. The script may pause and prompt you for a response at certain points before it
+     continues.
+  4. Wait for script to finish.
+
+## Make sure the deployment is up and ready
+
+  1. Check that all the components of the deployment are up in the Kubernetes
+     page. If everything is alright, everything should have a green check mark
+     in the console. Check everything under Cluster, Workloads and "Service &
+     Ingress".
+
+     Clusters will have "amppackager-cluster".
+     Workloads will have "user spec number of replicas of amppackager-cert-consumer-deployment,
+      amppackager-cert-renewer-pd and amppackager-nfs-server".
+     Service & Ingress will have "amppackager-nfs-service and
+      amppackager-service".
+
+  2. Issue curl command to check on the health of the amppackager. It should
+     return "ok".
+
+    curl http://AMP_PACKAGER_SERVICE_IP_ADDRESS:AMP_PACKAGER_SERVICE_PORT/healthz
+
+  3. Issue curl command to test if you can download the sample signed exchange
+     in https://amppackageexample.com:
+
+    curl -H 'Accept: application/signed-exchange;v=b3' -H 'AMP-Cache-Transform: google;v="1..2"' -i http://AMP_PACKAGER_SERVICE_IP_ADDRESS:AMP_PACKAGER_SERVICE_PORT/priv/doc/https://amppackageexample.com/
+
+## Optionally, run the teardown script (gcloud_down.sh)
```

Comment:
For the future, it would be good to think about what other health metrics we need (e.g. NFS free space, CPU, RAM per request...). I dunno what comes out of the box with gcloud or other platforms.

Reply:
Noted.